functional_tests.py

```diff
@@ -0,0 +1,18 @@
+from selenium import webdriver
+import unittest
+
+
+class NewVisitorTest(unittest.TestCase):
```

Comment:
You don't need to use unittest. Django has it's own test case that subclasses unittest

https://docs.djangoproject.com/en/2.0/topics/testing/overview/

Reply:
Right. So selenium is a seperate thing. Should still be doable with just django, for the future

https://lincolnloop.com/blog/introduction-django-selenium-testing/